ui/options_renaming.ui

```diff
@@ -222,6 +222,24 @@ p, li { white-space: pre-wrap; }
         </layout>
        </widget>
       </item>
+      <item>
+       <widget class="QLabel" name="ile_naming_format_documentation">
+        <property name="text">
+         <string><![CDATA[
+            <a href="https://picard.musicbrainz.org/docs/scripting">View full documentation online</a>
```

Comment:
Please use the same text as in https://github.com/musicbrainz/picard/blob/master/picard/ui/options/scripting.py#L140-L141 - again, to keep them the same.